### PR TITLE
feat: workflows icon

### LINKED ISSUE #712: [Feature]: Add workflows icon
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Adding the workflows icon to design system

### Why should this be worked on?

For completion of workflows project

### PR DESCRIPTION
**Pull request by ayushpahwa**
## Description

Added icon `node-tree` from figma for workflows.

Fixes #712 

Depends on # (pr)
> every PR in the design-system repository should have a corresponding PR in the appsmith repository to ensure that changes 
> here don't break existing flows. Link the corresponding PR here to trigger dpulls and prevent accidental merging.
> If relevant, link the enterprise PR here as well.

## Type of change

- New feature (non-breaking change which adds functionality)

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: packages/design-system-old/src/assets/icons/ads/workflows.svg
Comment (albinAppsmith) [inline packages/design-system-old/src/assets/icons/ads/workflows.svg:1]: Why is this icon added in old DS?

Comment (ayushpahwa) [reply packages/design-system-old/src/assets/icons/ads/workflows.svg:1]: Got confused while following old PRs, have updated the same now.

**Review by albinAppsmith — APPROVED** [review]

**Review by albinAppsmith — DISMISSED** [review]

**Review by albinAppsmith — APPROVED** [review]

**Pull request merged**